### PR TITLE
Solved #4 Add optional fields to Create Item 

### PR DESCRIPTION
**Pull request by vickivc**
Solved #4 Add optional fields to Create Item 
Also fixed issue of uploading receipt image

### CODE REVIEW

**Comment by subhankar-panda** [comment]
@vickivc  is this ready to merge?

**Comment by htoo97** [comment]
I'm caught up with stuff these days - i'll try to get to reviewing it next week!

File: src/server/api/controllers/admin/RequestController.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Get, JsonController, UseBefore, Res, Req, Post, Body } from 'routing-controllers';
+
+
+@JsonController('/request')
```
Comment (htoo97) [inline src/server/api/controllers/admin/RequestController.ts:4]: What is this for again?

Comment (subhankar-panda) [reply src/server/api/controllers/admin/RequestController.ts:4]: I think i pushed this by mistake, its for admins to interact with requests

**Pull request merged**